src/pages/docs/lexicon/image_card.md

```diff
@@ -0,0 +1,163 @@
+
+Start by adding the following imports to the imports section of your `init.jsp` file:
+
+```text/html
+// Import the clay tld file to be able to use the new tag
+<%@ taglib uri="http://liferay.com/tld/clay" prefix="clay" %>
+
+// Import the DropdownItem utility class to create the items model
+<%@ page import="com.liferay.frontend.taglib.clay.servlet.taglib.util.JSPDropdownItemList" %>
+```
+
+Don't forget to add the dependencies with to the `frontend-taglib-clay` and `fronteng.taglib.soy` module in your `build.gradle` file:
+
+```text/html
+provided group: "com.liferay", name: "com.liferay.frontend.taglib.soy", version: "1.0.0"
+
+provided project(":apps:foundation:frontend-taglib:frontend-taglib-clay")
+```
+
+- Url param is now href.
+- Text param is now title.
+- Icon is passed now trough icon param instead as a content with `horizontal-card-col` and `horizontal-card-col-icon`
+- For the actions you need to model them using the `JSPDropdownItemList` class.
+
+```text/html
+<clay:image-card
+	actionItems="<%=
+		new JSPDropdownItemList(pageContext) {
+			{
+				add(
+					dropdownItem -> {
+						dropdownItem.setHref(renderResponse.createRenderURL(), "mvcPath", "/edit_article.jsp");
+						dropdownItem.setLabel(LanguageUtil.get(request, "edit"));
+					}
+				);
+			}
+		}
+	%>"
+	href="<%= editURL %>"
+	imageSrc="<%= HtmlUtil.escape(articleImageURL) %>"
+	labels="<%=
+		new JSPLabelItemList(pageContext) {
+			{
+				final JournalArticle curArticle = (JournalArticle)result;
+
+				add(
+					labelItem -> {
+						labelItem.setLabel(LanguageUtil.get(request, WorkflowConstants.getStatusLabel(curArticle.getStatus())));
+						labelItem.setStyle(WorkflowConstants.getStatusCssClass(curArticle.getStatus()));
+					}
+				);
+			}
+		}
+	%>"
+	stickerLabel="<%= curArticleAuthor.getInitials() %>"
+	stickerShape="circle"
+	stickerStyle="<%= LexiconUtil.getUserColorCssClass(curArticleAuthor) %>"
+	subtitle="<%=
+		LanguageUtil.format(request, "x-modified-x-ago",
+			new String[] {
+				HtmlUtil.escape(curArticle.getUserName()),
+				LanguageUtil.getTimeDescription(request,
+					System.currentTimeMillis() - curArticle.getModifiedDate().getTime(),
+					true
+				)
+			}
+		)
+	%>"
+	title="<%= curArticle.getTitle(locale) %>"
+	resultRow="<%= row %>"
+	rowChecker="<%= articleSearchContainer.getRowChecker() %>"
+/>
+```
+
+Check out the following [Sample usage of JSPNavigationItemList](https://github.com/liferay/liferay-portal/commit/1ab9c006c750052f5c1c7df7072aab961b89966c) for more information about it.
+
+### [3] Move the action items definition into a Display Context <a id="step-3"></a>
+
+<div class="alert alert-info">This step is <em>optional</em> but <strong>recommended</strong></div>
+
+If your application already supports it (or even if it doesn't), consider moving the `Java` portion that generates the navigation items model into a `*DisplayContext` pattern to keep your jsps cleaner and server-free
+
+```text/html
+<clay:image-card
```

Comment:
We should have here the same params of the previous examples, and explain that labels should be moved too to a `*DisplayContext`